src/hooks/useDriveItem.ts

```diff
@@ -65,6 +76,8 @@ const useDriveItem = (props: UseDriveItemProps) => {
     thunk.then(() => {
       setIsDisabled(false);
     });
+
+    navigation.navigate('DrivePreview');
```

Comment:
Use enums for this instead of hardcoded strings. Therefore, in the future, if some alias, ID or something else changes, there is just one change required: on the enum

Reply:
This is typed, is not an enum but the argument is restricted to keyof RootStackParamList, which means you can use another string but TS will complain, making an enum for this doesnt make sense for me since this is already typed and has restricted values